chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -571,6 +639,18 @@ impl<'a, S: BlockchainStorageRead> TransactionVerifier<'a, S> {
         Ok(())
     }
 
+    fn remove_tokens(&mut self, tx: &Transaction) -> Result<(), ConnectTransactionError> {
+        let was_tokens_issued =
```

Comment:
Again, this is a consistency issue. It's IMPOSSIBLE to disconnect a block with token issuance without finding a token issued. Please don't hide such issues and handle them properly.

Reply:
I'll move this comment to where appropriate.